### PR TITLE
Dev in progress

### PR DESCRIPTION
**Pull request by Rehmsy**
Stuck on error. Connects to 27017 then seems to immediately disconnect and throw Error: connect ECONNREFUSED 127.0.0.1:80. Something in 2nd b4 each in tours.test seems to cause problem.

### CODE REVIEW

File: test/e2e/tours.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+};
+
+describe.skip('Tours API', () => {
+
+    beforeEach(() => dropCollection('tours'));
+
+    function save(tour) {
+        console.log('*****');
+        return request
+            .post('api/tours')
```
Comment (martypdx) [inline test/e2e/tours.test.js]: Need to include leading slash: `/api/tours`

File: .env [hunk 2]
```diff
@@ -0,0 +1,2 @@
+WU_API_KEY=5e1af6276eda1937
```
Comment (AMPotter) [inline .env]: Careful not to push .env

File: LAB.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ The schema structure of a tours looks like:
 path | type info
 ---|---
 title | required title of the tour
-activities | array of string activites that will happen during the show
+activities | array of string activities that will happen during the show
```
Comment (AMPotter) [inline LAB.md]: Haha, nice.

File: lib/app.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+app.use(express.json());
+
+//routes
+const tours = require('./routes/tours');
+
+app.use('/api/tours', tours);
+
+// 404 stuff goes below
+// including app handler last before module
+//const { handler, api404 } = require('./util/errors');
```
Comment (AMPotter) [inline lib/app.js]: This stuff probably shouldn't be commented out, but I understand this is probably to get Travis to pass so I can give a score.

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Juggalo Circus DB</title>
+</head>
+<body>
+    <h1>Dark Carnival DB</h1>
```
Comment (AMPotter) [inline public/index.html]: Whoop whoop

File: test/unit/tour.test.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
+const { assert } = require('chai');
+const { getErrors } = require('./helpers');
+const Tour = require('../../lib/models/tour');
+
+describe('Tour model', () => {
+
+    it('validates good model', () => {
+        const data = {
+            title: 'Jezbo\'s Dark Carnival',
+            activities: ['Hatchet Juggling', 'Dubstep Finger Tutting', 'Faygo Contest', 'Axe Toss', 'Joker Card Tarot', 'Carnival of Terrier Dog Wash'],
```
Comment (AMPotter) [inline test/unit/tour.test.js]: Looks like someone's down with the clown

**Pull request merged**